bin/pipeline_result_processor/prp/cli.py

```diff
@@ -0,0 +1,81 @@
+import click
+import logging
+from logging.config import dictConfig
+import pandas as pd
+import json
+from .parse import parse_resistance_pred, parse_virulence_pred, parse_qust_results, parse_cgmlst_results, parse_mlst_results
+
+dictConfig(
+    {
+        "version": 1,
+        "disable_existing_loggers": False,
+        "formatters": {
+            "default": {
+                "format": "[%(asctime)s] %(levelname)s in %(module)s: %(message)s",
+            }
+        },
+        "root": {"level": "INFO"},
+    }
+)
+LOG = logging.getLogger(__name__)
+
+
+SPP_MIN_READ_FRAC = 0.001
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.command()
+@click.option('-i', '--sample-id', required=True, help="Sample identifier")
+@click.option('-u', '--run-metadata', type=click.File(), required=True, help="Analysis metadata from pipeline")
+@click.option('-q', '--quast', type=click.File(), help="Quast quality control metrics")
+@click.option('-p', '--process-metadata', type=click.File(), multiple=True, help="Meta data from nextflow processes")
+@click.option('-k', '--kraken', type=click.File(), help="Kraken specie annotation results")
+@click.option('-m', '--mlst', type=click.File(), help="MLST prediction results")
+@click.option('-c', '--cgmlst', type=click.File(), help="cgMLST prediction results")
+@click.option('-v', '--virulence', type=click.File(), help="Virulence factor prediction results")
+@click.option('-r', '--resistance', type=click.File(), help="Resistance prediction results")
+@click.argument('output', type=click.File('w'))
+def create_output(sample_id, run_metadata, quast, process_metadata, kraken, mlst, cgmlst, virulence, resistance, output):
```

Comment:
Could also use some variation of kwargs

Reply:
Sure. I just think this is more readable